src/PaymentGateways/ServiceProvider.php

```diff
@@ -65,12 +66,14 @@ public function boot()
     }
 
     /**
+     * @unreleased add CopyV2GatewaysSettingsToV3 migration
      * @since 2.33.0 add RemovePaymentIntentSecretMeta migration
      * @since 2.19.6
      */
     private function registerMigrations()
     {
         give(MigrationsRegister::class)->addMigrations([
+            CopyV2GatewaysSettingsToV3::class,
```

Comment:
I think the migrations runs in order - so I would put this last in the list.

Reply:
Updated on 707e8bbff7a529210ccbc63e2f80ab42756625fe